tests/unit/test_tb_x86.c

```diff
@@ -222,7 +222,7 @@ static void test_tb_x86_64_32_imul_Gv_Ev_Ib(void **state)
 {
     uc_engine *uc = *state;
     uc_hook trace1, trace2;
-    void *mem;
+    //void *mem;
```

Comment:
What is this for ? If we exclude this test, let's leave this out from this commit.